mlir/test/Conversion/TosaToMLProgram/tosa-to-mlprogram.mlir

```diff
@@ -0,0 +1,16 @@
+// RUN: mlir-opt --split-input-file --tosa-to-mlprogram %s -verify-diagnostics -o -| FileCheck %s
```

Comment:
Lets drop --split-input-file -verify-diagnostics and line 4 here, will result in nicer links to failed CHECKs and there are no errors being verified here.

Reply:
done.